configure/os/CONFIG.linux-arm.linux-arm

```diff
@@ -6,3 +6,8 @@
 
 # Include common gnu compiler definitions
 include $(CONFIG)/CONFIG.gnuCommon
+
+STATIC_LDFLAGS_YES= -Wl,-Bstatic
+STATIC_LDFLAGS_NO=
+STATIC_LDLIBS_YES= -Wl,-Bdynamic
+STATIC_LDLIBS_NO=
```

Comment:
For reference, this change is overriding:

https://github.com/epics-base/epics-base/blob/91941af992f6c32ef4f4ef55a2952f581aeee36a/configure/CONFIG.gnuCommon#L54-L55

I'm wondering why the other definition in linuxCommon isn't already doing this.

@anjohnson Any idea?  Is this an include ordering issue effecting other `linux-*` targets?

https://github.com/epics-base/epics-base/blob/91941af992f6c32ef4f4ef55a2952f581aeee36a/configure/os/CONFIG.Common.linuxCommon#L22-L24

Reply:
Having found 3 other re-definitions for `linux-*` targets, I guess the (re)definition in linuxCommon has no effect?  Should it be removed?  (to marginally reduce confusion)

https://github.com/epics-base/epics-base/blob/91941af992f6c32ef4f4ef55a2952f581aeee36a/configure/os/CONFIG.linux-x86.linux-arm#L24-L27

https://github.com/epics-base/epics-base/blob/91941af992f6c32ef4f4ef55a2952f581aeee36a/configure/os/CONFIG.linux-x86.linux-x86#L10-L13

https://github.com/epics-base/epics-base/blob/91941af992f6c32ef4f4ef55a2952f581aeee36a/configure/os/CONFIG.linux-x86_64.linux-aarch64#L24-L27

Reply:
... Now that I remember `show-makefiles`, I can answer my own question.  The definitions in linuxCommon are never used.  (I keep forgetting how late the gnuCommon snippet is included.)

So maybe a better fix would be to use `?=` for the definitions in gnuCommon?

```
$ make -C configure/O.linux-x86_64 show-makefiles
...
Makefiles read:
...
    ../../configure/os/CONFIG.linux-x86_64.Common
    ../../configure/os/CONFIG.UnixCommon.Common
    ../../configure/os/CONFIG_SITE.linux-x86_64.Common
    ../../cfg/TOOLCHAIN.linux-x86_64.linux-x86_64
    ../../configure/os/CONFIG.Common.linux-x86_64
    ../../configure/os/CONFIG.Common.linuxCommon
    ../../configure/os/CONFIG.Common.UnixCommon
    ../../configure/os/CONFIG_SITE.linux-x86_64.UnixCommon
    ../../configure/os/CONFIG_SITE.Common.linuxCommon
    ../../configure/os/CONFIG.linux-x86_64.linux-x86_64
    ../../configure/os/CONFIG.linux-x86.linux-x86
    ../../configure/CONFIG.gnuCommon
    ../../configure/os/CONFIG_SITE.Common.linux-x86_64
    ../../configure/os/CONFIG_SITE.linux-x86_64.linux-x86_64

...
```